bokehjs/src/coffee/models/glyphs/xy_glyph.ts

```diff
@@ -27,16 +27,14 @@ export namespace XYGlyph {
     x: NumberSpec
     y: NumberSpec
   }
-
-  export interface Opts extends Glyph.Opts {}
 }
 
 export interface XYGlyph extends XYGlyph.Attrs {}
 
 export abstract class XYGlyph extends Glyph {
 
-  constructor(attrs?: Partial<XYGlyph.Attrs>, opts?: XYGlyph.Opts) {
-    super(attrs, opts)
+  constructor(attrs?: Partial<XYGlyph.Attrs>) {
+    super(attrs)
   }
```

Comment:
If `constructor` only calls `super` and nothing else, is it needed at all?

Reply:
It's needed, because it asserts its type. Otherwise you will be using the type of the parent constructor, and in general `HasProps`', which allows any attributes. It TypeScript had a stronger type system (supported class type members), then this may not be needed.